lms/djangoapps/instructor/enrollment.py

```diff
@@ -361,6 +361,9 @@ def _reset_module_attempts(studentmodule):
     problem_state = json.loads(studentmodule.state)
     # old_number_of_attempts = problem_state["attempts"]
     problem_state["attempts"] = 0
+    problem_state["score_history"] = []
+    problem_state["correct_map_history"] = []
```

Comment:
Why isn't the correct_map reset as well?

Reply:
Because we want to reset the attempts to zero, but we don't want to delete the current score status.

Reply:
I'm not sure I follow. What do you mean by _current score status_?

Reply:
When we use the **Reset Learner's Attempts to Zero** functionality, we only want to reset the attempts, but not the score. Therefore, the fields related to the current score, such as the correct_map field, should not be modified.

Reply:
Thanks!